include/tinystl/unordered_map.h

```diff
@@ -69,22 +72,23 @@ namespace tinystl {
 
 	private:
 
+		void rehash(size_t nbuckets);
+
 		typedef unordered_hash_node<Key, Value>* pointer;
 
 		size_t m_size;
-		buffer<pointer, Alloc> m_buckets;
+		tinystl::buffer<pointer, Alloc> m_buckets;
 	};
 
 	template<typename Key, typename Value, typename Alloc>
-	unordered_map<Key, Value, Alloc>::unordered_map()
+	inline unordered_map<Key, Value, Alloc>::unordered_map()
 		: m_size(0)
 	{
 		buffer_init<pointer, Alloc>(&m_buckets);
-		buffer_resize<pointer, Alloc>(&m_buckets, 9, 0);
```

Comment:
@bkaradzic The crash you were seeing is due to this line being removed. I found the offending commit:

https://github.com/mendsley/tinystl/commit/2e13dfc3eaa600c2bc5d3c5c6e5a11095846fd1f